src/utils/entries/entries.ts

```diff
@@ -0,0 +1,5 @@
+import { Entries } from './entries.types';
+
+export function entries<TEntries>(obj: TEntries): Entries<TEntries> {
+  return Object.entries(obj) as any;
```

Comment:
Yope. Czy dodałeś niemożność zakomitowania `any` do eslinta? ;)

Reply:
Teraz dodałem. W piątek rzuciłem to rozwiązanie z any, bo znalazłem je na stacOverflow, pobawiłem się nim chwilę i stwierdziłem, że to przypadek, w którym właśnie any jest nieuniknione, ale dzisiaj na świeżo sobie to już ogarnąłem bez any, a samo any, jak wyżej, zbanowałem.